mithril.js

```diff
@@ -218,9 +218,34 @@
 		return list
 	}
 
+	var applyIEPerf = (
+			typeof document !== "undefined" &&
+			typeof document.documentMode === "number"
+		) ||
+		(
+			typeof navigator !== "undefined" &&
+			typeof navigator.userAgent === "string" &&
+			/\bEdge\/\d/.test(navigator.userAgent)
+		)
+
 	function insertNode(parentElement, node, index) {
-		parentElement.insertBefore(node,
-			parentElement.childNodes[index] || null)
+		if (applyIEPerf) {
+			if (!parentElement._children) {
+				parentElement._children = []
+			}
+			parentElement._children.push(node)
```

Comment:
[This line](https://github.com/lhorie/mithril.js/pull/1016/files#diff-0bf87cbdb33b829b275ac7973ba7bc1bR246) doesn't mirror the other implementation of `parentElement.insertBefore(node, parentElement.childNodes[index] || null)`.

That's why I'm skeptical. I'm running Linux, so I can't exactly test this readily, but that part is sending off alarms in my head.